apps/aecore/test/aecore_sync_SUITE.erl

```diff
@@ -991,7 +991,7 @@ make_shortcut(Config) ->
     ok = filelib:ensure_dir(filename:join(PrivDir, "foo")),
     Shortcut = shortcut_dir(Config),
     delete_file(Shortcut),
-    ok = file:make_symlink(PrivDir, Shortcut),
+    symlink(PrivDir, Shortcut),
```

Comment:
There will be two entries in the Common Test log for this symlink, is that intentional?